webpack.config.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  mode: 'production',
+  devtool: 'source-map',
+  entry: {
+    application: './app/javascript/application.js',
+  },
+  output: {
+    filename: '[name].js',
+    sourceMapFilename: '[file].map',
+    path: path.resolve(__dirname, 'app/assets/builds'),
+  },
+  plugins: [
+    new webpack.optimize.LimitChunkCountPlugin({
+      maxChunks: 1,
+    }),
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+        },
+      },
+    ],
+  },
+};
```

Comment:
- [ ] As pointed out above :arrow_up_small: let's just copy and paste config files as requested since its a requirements. Let's always make sure our clients requirements are fulfill before opting for any align requirement :+1: